src/index.html

```diff
@@ -22,6 +22,347 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <div>
+      <header class="header">
+        <a href="#">
+          <img
+            class="img"
+            src="/src/images/logo.png"
+            alt="MOYO logo"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                class="nav__link is-active"
+                href="#"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Samsung
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Smartphones
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                data-qa="nav-hover"
+                href="#"
+              >
+                Gadgets
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Tablets
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Photo
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+    </div>
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
```

Comment:
The first card should have a `data-qa="card"` attribute as per the task requirements.